app/controllers/website/search_controller.rb

```diff
@@ -1,8 +1,15 @@
 class Website::SearchController < WebsiteController
   def index
-    @pagy, @books = pagy Book.search_book_name(params[:book_name])
-                             .search_author(params[:author])
-                             .search_category(params[:category]),
-                         items: Settings.book.max_page
+    @pagy, @books = pagy Book.all,
+                         items: Settings.book.max_page,
+                         link_extra: 'data-remote="true"'
+    @q = Book.ransack(params[:q])
```

Comment:
thay @q bằng từ khác dc k e?

Reply:
ở đây đã search gì chưa e? c thấy e trả về toàn bộ danh sách book

Reply:
thay @q bằng từ khác dc k e?
đc ạ
ở đây đã search gì chưa e? c thấy e trả về toàn bộ danh sách book
chỗ này e sửa r ạ

Reply:
ố gắng áp dụng càng nhiều càng tốt nhé. phần get started
dạ vâng c